development/build/scripts.js

```diff
@@ -1214,11 +1214,18 @@ function renderHtmlFile({
   const htmlTemplate = readFileSync(htmlFilePath, 'utf8');
 
   const eta = new Eta();
-  const htmlOutput = eta.renderString(htmlTemplate, {
-    isMMI,
-    isTest,
-    shouldIncludeSnow,
-  });
+  const htmlOutput = eta
+    .renderString(htmlTemplate, { isMMI, isTest, shouldIncludeSnow })
+    // these replacements are added to support the webpack build's automatic
+    // compilation of html files, which the gulp-based process doesn't support.
+    .replace('./scripts/load/background.ts', './load-background.js')
+    .replace(
+      '<script src="./load-background.js" defer></script>',
+      '<script src="./load-background.js" defer></script>\n    <script src="./chromereload.js" defer></script>',
+    )
+    .replace('./scripts/load/ui.ts', './load-app.js')
+    .replace('../ui/css/index.scss', './index.css')
+    .replace('@lavamoat/snow/snow.prod.js', './scripts/snow.js');
```

Comment:
The webpack build automatically finds and includes resources in our HTML files (and manifest.json), but the gulp-build handles everything manually. The webpack build needs our HTML files to point to the location of the files _before_ bundling, so this gross bits here fix up the HTML files for the gulp build.